proto/brpb.proto

```diff
@@ -190,6 +190,14 @@ message BackupRequest {
     int32 compression_level = 13;
     // The cipher_info is Used to encrypt sst
     CipherInfo cipher_info = 14;
+
+    // dst_api_version indicates the key-value encoding version used by the
+    // generated SST file. Accepted values:
+    // 1. "v1": the generated SST files are encoded with api-v1, can be restored
```

Comment:
"v1" can still be used on clusters above "v6.0". And the first version supporting "v2" would be "v6.1". Maybe it would be better to not mention the version.

Reply:
done